openedx/features/course_experience/views/course_home.py

```diff
@@ -89,25 +87,35 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         """
         course_key = CourseKey.from_string(course_id)
 
-        # Render the outline as a fragment
-        outline_fragment = CourseOutlineFragmentView().render_to_fragment(request, course_id=course_id, **kwargs)
-
-        # Get resume course information
-        has_visited_course, resume_course_url = self._get_resume_course_info(request, course_id)
-
-        # Render the welcome message as a fragment
-        welcome_message_fragment = WelcomeMessageFragmentView().render_to_fragment(
-            request, course_id=course_id, **kwargs
-        )
-
         # Render the course dates as a fragment
         dates_fragment = CourseDatesFragmentView().render_to_fragment(request, course_id=course_id, **kwargs)
 
         # TODO: Use get_course_overview_with_access and blocks api
```

Comment:
I know you didn't add this comment, but I'd remove the prefix `TODO`, and either remove this comment entirely, or update it to read something like:
```
Cannot use get_course_overview_with_access unless XXX.
```
What is XXX?  Course updates are moved into xblocks?  Other?  If we can't provide anything useful, than just remove altogether.